plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityInterpolationDuration.java

```diff
@@ -0,0 +1,45 @@
+package com.denizenscript.denizen.objects.properties.entity;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.DurationTag;
+import org.bukkit.entity.Display;
+
+public class EntityInterpolationDuration extends EntityProperty<DurationTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name interpolation_duration
+    // @input DurationTag
+    // @description
+    // The duration a display entity will spend interpolating between interpolated value(s).

```

Comment:
The interpolation things should have `<@link`s between them, and the "can be interpolated" meta should `<@link` somewhere to explain it. Maybe a short language meta as a centerpoint? Either for interpolation specifically, or as a general directory of info about display entities.

Reply:
Added an interpolation specific meta and linked it for now, let me know if it should be expanded